src/plots/cartesian/axis_defaults.js

```diff
@@ -103,8 +116,8 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, coerce,
 
     if(showLine || containerOut.ticks) coerce('mirror');
 
-    var gridColor = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'gridcolor'),
-        gridWidth = Lib.coerce2(containerIn, containerOut, layoutAttributes, 'gridwidth'),
+    var gridColor = coerce2('gridcolor', lightColor(dfltColor, options.bgColor)),
```

Comment:
> I think that's an improvement as a default, do you agree?

Agreed. But maybe you could mention this blending behavior in the `axis.color` [description](https://github.com/plotly/plotly.js/pull/390/files#diff-0e41c2e162564438ff091d0ed6b5b455R21) ?